app/models/patient.rb

```diff
@@ -227,6 +228,10 @@ def initialize_fulfillment
     build_fulfillment.save
   end
 
+  def update_handling_fund
+    self.solidarity = false if self.handling_fund.blank?
```

Comment:
I kinda think we can be a little lazier here and not have this autotoggle - I don't want to be too prescriptive about the use at first.